packages/form-footer/src/FormFooter.tsx

```diff
@@ -58,6 +58,16 @@ const buttonStyle = css`
 /**
  * Types
  */
+interface BaseButtonProps {
+  text?: string;
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
+}
```

Comment:
Should we extend `ButtonProps` from the button package?

Reply:
good point, I can use that instead. I will need to make `leftGlyph` nullable though